File: C#/3346.cs
using System; 
class URI {

    static void Main(string[] args) { 
        double f1, f2, ft;

        string[] s = Console.ReadLine().Split(' ');
        f1 = double.Parse(s[0]);
        f2 = double.Parse(s[1]);

        ft = ((1 + f1 / 100) * (1 + f2 / 100) - 1) * 100;

        Console.WriteLine(ft.ToString("F6"));
    }
}

File: Dart/3346.dart
import 'dart:io';

void main() {
    double f1, f2, ft;

    String? input = stdin.readLineSync();
    List<String> s = input != null ? input.split(' ') : [];
    f1 = double.parse(s[0]);
    f2 = double.parse(s[1]);

    ft = ((1 + f1 / 100) * (1 + f2 / 100) - 1) * 100;

    print(ft.toStringAsFixed(6));
}